modules/apps/29-fee/transfer_test.go

```diff
@@ -65,7 +65,7 @@ func (suite *FeeTestSuite) TestFeeTransfer() {
 	)
 
 	suite.Require().Equal(
-		fee.AckFee.Add(fee.TimeoutFee...), // ack fee paid, timeout fee refunded
+		fee.AckFee, // ack fee paid, no refund needed since timeout_fee = recv_fee + ack_fee
```

Comment:
i find this comment slightly confusing, it's more that the timeout fee wasn't escrowed since it was smaller than the recv+ack, correct?

Reply:
In this case, there is equality due to hard-coded values